lib/types/entry.ts

```diff
@@ -175,6 +175,11 @@ export type GenericEntryCollection<Fields extends FieldsType> = AbstractEntryCol
   GenericEntry<Fields>
 >
 
+export type GenericEntryCollectionWithAllLocales<
```

Comment:
Is there a reason to introduce a new custom entry type?

Reply:
I removed it (but added different ones)